src/main/java/name/valery1707/kaitai/KaitaiMojo.java

```diff
@@ -195,18 +200,18 @@ private void executeInt() throws KaitaiException {
 		sourceDirectory = sourceDirectory.toPath().normalize().toFile();
 		if (!sourceDirectory.exists()) {
 			getLog().warn(format(
-				"Skip KaiTai generation: Source directory does not exists: %s"
-				, sourceDirectory.toPath().normalize().toFile().getAbsolutePath()
+					"Skip KaiTai generation: Source directory does not exists: %s"
+					, sourceDirectory.toPath().normalize().toFile().getAbsolutePath()
```

Comment:
Please revert the indentation changes
```suggestion
				"Skip KaiTai generation: Source directory does not exists: %s"
				, sourceDirectory.toPath().normalize().toFile().getAbsolutePath()
```